File: basic/Output.cs
using System;

public class Output
{
  public static void Main()
  {
    /*
      Polimorfismo da função Console.WriteLine
    */
    int output_int       = 1;
    char output_char     = '$';
    float output_float   = 1.2f;
    double output_double = 1.2;
    string output_string = "string";

    Console.WriteLine(output_int);
    Console.WriteLine(output_char);
    Console.WriteLine(output_float);
    Console.WriteLine(output_double);
    Console.WriteLine(output_string);
  }
}

File: basic/Output.ex
defmodule Output do
  def main do
    output_int = 1
    output_char = '$'
    output_float = 1.2
    output_string = "string"

    IO.puts output_int
    IO.puts output_char
    IO.puts output_float
    IO.puts output_string
  end
end

Output.main
